### ISSUE TITLE
Clarify permission object properties

### ISSUE THREAD

**Issue by alexandratran** [MetaMask/api-specs#278]
The spec describes the [permission object](https://github.com/MetaMask/api-specs/blob/main/openrpc.yaml#L1048-L1072) to have the following properties:

- `id`
- `'@context'`
- `invoker`
- `caveats`

But, when testing the methods in the interactive reference, I'm getting the following properties in the result:

- `id`
- `parentCapability`
- `invoker`
- `caveats`
- `date`

Can we update the spec to reflect the latter?

**Comment by jiexi** [comment]
Yes, please update the spec to match what you see returned. The spec is out of date

**Comment by jiexi** [comment]
Thank you for catching this and fixing it!
